wordpress-mobile/WordPress-Android
# Fix design issues with Promo Dialog

Fixes #5802. Thanks a lot @nbradbury for your help! You've pretty much fixed the whole thing. Do you mind reviewing it as well when you have the time, since you already know what the changes are.

The background color change to white has been blessed by @mattmiklic :)

To test:
* Open stats promo dialog for pre and post lollipop to make sure it looks good.
* You can ignore the visual editor one since removing that will be my next PR #5819 

**Post-lollipop**
![screenshot_1493737769](https://cloud.githubusercontent.com/assets/662023/25660274/2ee0939e-3014-11e7-93cc-e8225acc6849.png)

**Pre-lollipop**
![screenshot_1493814098](https://cloud.githubusercontent.com/assets/662023/25660300/46eb0d2a-3014-11e7-8d66-e94a27dfac44.png)
